docs.phtml

```diff
@@ -31,25 +31,25 @@
     <li><a href="<?=$top_dir;?>/viewdoc.phtml?rev=<?=$major_stable;?>&amp;lang=C&amp;doc=guide"><b><?php echo T_("The Tutorial and Concepts Guide");?></b></a></li>
   </ul>
 
-  <p class="justify"><?php echo T_("The <b>Help Manual</b> is designed to be a quick reference of how to accomplish specific tasks and how to use the features in <span class=\"gnucash\">GnuCash</span>. The <b>Concepts Guide</b> is designed to be an in depth guide to the concepts behind using <span class=\"gnucash\">GnuCash</span> with a tutorial to show how to put those concepts into practice.");?></p>
+  <p class="justify"><?php echo T_("The <b>Help Manual</b> details specific tasks and how to use features <span class=\"gnucash\">GnuCash</span>. The <b>Concepts Guide</b> is an in-depth guide to <span class=\"gnucash\">GnuCash</span> tutorial.");?></p>
 
   <p class="justify"><?php
     /* Todo: menu.phtml has a similar link. They should be unified with the other language stuff.*/
     /* Translators: If wiki pages in your laguage exists, consider to replace "De"/"Deutsch" by your language.*/
-    echo T_("Other resources are the <a href=\"https://wiki.gnucash.org/wiki/\">English</a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class=\"gnucash\">GnuCash</span> wikis.");
+    echo T_("Other resources are to be found in the <a href=\"https://wiki.gnucash.org/wiki/\">English</a> or <a href=\"https://wiki.gnucash.org/wiki/De/\">Deutsch</a> <span class=\"gnucash\">GnuCash</span> wikis.");
     ?></p>
 
   <p class="justify"><?php
-  echo T_("You can also send an email to the <a href=\"mailto:gnucash-user@gnucash.org\">gnucash-user</a> mailing list if you cannot find a satisfactory answer to your question within either the <b>Help Manual</b> or the <b>Tutorial and Concepts Guide</b>. We <b>want</b> feedback from you, it is also through your comments that we know how to improve the documentation.");
+  echo T_("Send an e-mail to the <a href=\"mailto:gnucash-user@gnucash.org\">gnucash-user</a> mailing list if you have tried the <b>Help Manual</b> and the <b>Tutorial and Concepts Guide</b>. Feedback and hints on what resources are missing is greatly appreciated.");
```

Comment:
Prefer (with appropriate tags):

Send an e-mail to the gnucash-user mailing list if you have tried the Help Manual and the Tutorial and Concepts Guide without finding an answer. Your contribution of additional feedback and hints is also greatly appreciated.

Reply:
Prefer "email" to "e-mail".

Reply:
e-mail has more errors in translation, and I'll for once champion the proper variant, even though I considered using it as a verb. :)

Reply:
Using "email" as a verb is common usage, so IMO "Email the gnucash-user mailing list" would be fine.